sql/moz-fx-data-shared-prod/stripe_derived/subscriptions_v1/metadata.yaml

```diff
@@ -11,5 +11,3 @@ scheduling:
   # destination is the whole table, not a single partition,
   # so don't use date_partition_parameter
   date_partition_parameter:
-  referenced_tables: [['moz-fx-data-shared-prod', 'stripe_external',
-                       'subscriptions_v1']]
```

Comment:
these are left over from a time before referenced tables could be determined on queries that can't be dry-run in CI